<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGameSettingsDialog</class>
 <widget class="QDialog" name="QGameSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game Settings Manager</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Lives</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Init value:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="livesInitSpin">
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Max value:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="livesMaxSpin">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Icon:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="livesCombo">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="livesButton">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Keys</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Init value:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="keysInitSpin">
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Max value:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="keysMaxSpin">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Icon:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="keysCombo">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="keysButton">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1" rowspan="2">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Levels</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>Editor</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QListWidget" name="editorLevelList"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QPushButton" name="addLevelButton">
            <property name="maximumSize">
             <size>
              <width>28</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="S2PEditor.qrc">
              <normaloff>:/Common/assets/images/GUI_Icons/add-1.png</normaloff>:/Common/assets/images/GUI_Icons/add-1.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>Game</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_18">
        <item>
         <widget class="QListWidget" name="gameLevelList"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QPushButton" name="levelUpButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="S2PEditor.qrc">
              <normaloff>:/Common/assets/images/GUI_Icons/arrowUp.png</normaloff>:/Common/assets/images/GUI_Icons/arrowUp.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="levelDownButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="S2PEditor.qrc">
              <normaloff>:/Common/assets/images/GUI_Icons/arrowDown.png</normaloff>:/Common/assets/images/GUI_Icons/arrowDown.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteLevelButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="S2PEditor.qrc">
              <normaloff>:/QObjectsDock/assets/images/GUI_Icons/error.png</normaloff>:/QObjectsDock/assets/images/GUI_Icons/error.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="idLabel">
        <property name="text">
         <string>&lt;No item selected&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sourceLine"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox_7">
     <property name="title">
      <string>Game window</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="minimumSize">
           <size>
            <width>31</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Width</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="windowWidthSpin">
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QLabel" name="label_18">
          <property name="minimumSize">
           <size>
            <width>31</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Height</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="windowHeightSpin">
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
     <zorder></zorder>
     <zorder></zorder>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>Screens</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="startCombo">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="startScreenButton">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Pause</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="pauseCombo">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pauseScreenButton">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>GameOver</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="gameOverCombo">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="gameOverScreenButton">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>End</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="endCombo">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="endScreenButton">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Health</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Init value:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="healthInitSpin">
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Max value:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="healthMaxSpin">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Icon:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="healthCombo">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="healthButton">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Score</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Init value:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="scoreInitSpin">
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Max value:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="scoreMaxSpin">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Icon:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="scoreCombo">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="scoreButton">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>healthInitSpin</tabstop>
  <tabstop>healthMaxSpin</tabstop>
  <tabstop>healthCombo</tabstop>
  <tabstop>healthButton</tabstop>
  <tabstop>livesInitSpin</tabstop>
  <tabstop>livesMaxSpin</tabstop>
  <tabstop>livesCombo</tabstop>
  <tabstop>livesButton</tabstop>
  <tabstop>scoreInitSpin</tabstop>
  <tabstop>scoreMaxSpin</tabstop>
  <tabstop>scoreCombo</tabstop>
  <tabstop>scoreButton</tabstop>
  <tabstop>keysInitSpin</tabstop>
  <tabstop>keysMaxSpin</tabstop>
  <tabstop>keysCombo</tabstop>
  <tabstop>keysButton</tabstop>
  <tabstop>startCombo</tabstop>
  <tabstop>startScreenButton</tabstop>
  <tabstop>pauseCombo</tabstop>
  <tabstop>pauseScreenButton</tabstop>
  <tabstop>gameOverCombo</tabstop>
  <tabstop>gameOverScreenButton</tabstop>
  <tabstop>endCombo</tabstop>
  <tabstop>endScreenButton</tabstop>
  <tabstop>windowWidthSpin</tabstop>
  <tabstop>windowHeightSpin</tabstop>
  <tabstop>gameLevelList</tabstop>
  <tabstop>levelUpButton</tabstop>
  <tabstop>levelDownButton</tabstop>
  <tabstop>sourceLine</tabstop>
 </tabstops>
 <resources>
  <include location="S2PEditor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QGameSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>521</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QGameSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>517</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>518</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
